cli/start.go

```diff
@@ -147,6 +134,13 @@ func MakeStartCommand() *cobra.Command {
 			isDevMode := cfg.GetBool("development")
 			if isDevMode {
 				cmd.Printf(devModeBanner)
+				if cfg.GetBool("keyring.disabled") {
```

Comment:
note: If keyring is available we use it for the identity. Otherwise we generate a temporary one only if `--development` flat is passed. I thought temporary identities make sense only if you explicitly in non-production mode. Not 100% sure we want it like this.

Reply:
This should definitely be documented in the `README`  specially near the `keyring` section

Reply:
done